packages/grafana-ui/src/saga-themes/code/core/spacing.ts

```diff
@@ -0,0 +1,21 @@
+export default {
+  dimension: {
+    scale: '8px',
+  },
+  spacing: {
+    x0: '0px',
+    x25: '2px',
+    x50: '4px',
+    x75: '6px',
+    x100: '8px',
+    x150: '12px',
+    x200: '16px',
+    x250: '20px',
+    x300: '24px',
+    x400: '32px',
+    x500: '40px',
+    x600: '48px',
+    x800: '64px',
+    x1000: '80px',
+  },
```

Comment:
@Aires-Grafana on top of the spacing tokens, are we considering adding sizing tokens, used to specify width/height of the elements? I wonder if there's something like this on the design side already?

Reply:
@Clarity-89 Yes, this was the first scale added because it already exists, but the goal for the token structure is to have all related layout tokens added as well, sizing, dimensions, breakpoints, and content min and max widths. All are open for discussion and to be added.